freckle-app.cabal

```diff
@@ -23,6 +23,11 @@ source-repository head
   type: git
   location: https://github.com/freckle/freckle-app
 
+flag git
```

Comment:
I might call this `test-git` or something? A flag just named `git` might imply you are disabling all git functionality, not just the testing thereof.